tcms/templates/setup.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Initialize database" %}{% endblock %}
+{% block body_class %}cards-pf{% endblock %}
+
+{% block contents %}
+    <div class="container-fluid container-cards-pf">
+        <div class="row row-cards-pf">
+            <div class="col-sm-12 col-md-12 col-lg-12">
+                <div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-aggregate-status-mini">
+                    <h2 class="card-pf-title">
+                        <span class="fa fa-database"></span>
+                        <span class="card-pf-aggregate-status-count">
+                            {% trans "Initialize database" %}
+                        </span>
+                        <br>
+                        {% trans "Your database has not been initialized yet."%} &nbsp;
+                        {% trans "Click the button below to initialize it!" %}
```

Comment:
Both sentences should be translated together.

Reply:
Add: "WARNING: this operation will take a while! This page will redirect when done."